### PR TITLE
aircrack-ng: add German translation

### PR DESCRIPTION
**Pull request by Slotharmy**
Translated the current aircrack-ng.md into german.

- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/aircrack-ng.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# aircrack-ng
+
+> WEP und WPA/WPA2 Schlüssel im Kommunikationsaufbau knacken.
+> Teil des Aircrack-ng Softwarepakets.
+> Davor muss Packet abgefangen werden mit aircrack
+> Mehr Informationen: <https://www.aircrack-ng.org/doku.php?id=aircrack-ng> (auf Englisch)
```
Comment (kbdharun) [inline pages.de/common/aircrack-ng.md:6]:
```suggestion
> Mehr Informationen: <https://www.aircrack-ng.org/doku.php?id=aircrack-ng>.
```

File: pages.de/common/aircrack-ng.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# aircrack-ng
+
+> WEP und WPA/WPA2 Schlüssel im Kommunikationsaufbau knacken.
+> Teil des Aircrack-ng Softwarepakets.
+> Davor muss Packet abgefangen werden mit aircrack
```
Comment (kbdharun) [inline pages.de/common/aircrack-ng.md:5]:
```suggestion
```
Idk, if this line is required, I will wait for a native speakers opinion.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages.de/common/aircrack-ng.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+> Mehr Informationen: <https://www.aircrack-ng.org/doku.php?id=aircrack-ng> (auf Englisch)
+
+- Schlüssel knacken von abgefangenen Packeten mithilfe von Wortlisten:
+
+`aircrack-ng -w {{Pfad/zu/Wortliste.txt}} {{Pfad/zu/Packetdatei.cap}}`
```
Comment (blueskyson) [inline pages.de/common/aircrack-ng.md:10]:
```suggestion
`aircrack-ng -w {{pfad/zu/wortliste.txt}} {{pfad/zu/packetdatei.cap}}`
```

File: pages.de/common/aircrack-ng.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+- Schlüssel knacken von abgefangenen Packeten mithilfe einer Wortliste und der (E)SSID des Access Points:
+
+`aircrack-ng -w {{Pfad/zu/Wortliste.txt}} -e {{essid}} {{Pfad/zu/Packetdatei.cap}}`
```
Comment (blueskyson) [inline pages.de/common/aircrack-ng.md:14]:
```suggestion
`aircrack-ng -w {{pfad/zu/wortliste.txt}} -e {{essid}} {{pfad/zu/packetdatei.cap}}`
```

File: pages.de/common/aircrack-ng.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+- Schlüssel knacken von abgefangenen Packeten mithilfe einer Wortliste und der MAC-Adresse des Access Points:
+
+`aircrack-ng -w {{Pfad/zu/Wortliste.txt}} --bssid {{mac}} {{Pfad/zu/Packetdatei.cap}}`
```
Comment (blueskyson) [inline pages.de/common/aircrack-ng.md:18]:
```suggestion
`aircrack-ng -w {{pfad/zu/wortliste.txt}} --bssid {{mac}} {{pfad/zu/packetdatei.cap}}`
```

File: pages.de/common/aircrack-ng.md (hunk 2 above)
Comment (kbdharun) [inline pages.de/common/aircrack-ng.md:5]:
```suggestion
```

File: pages.de/common/aircrack-ng.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# aircrack-ng
+
+> WEP und WPA/WPA2 Schlüssel im Kommunikationsaufbau knacken.
+> Teil des Aircrack-ng Softwarepakets.
+> Weitere Informationen: <https://www.aircrack-ng.org/doku.php?id=aircrack-ng>.
+
+- Schlüssel knacken von abgefangenen Packeten mithilfe von Wortlisten:
```
Comment (pixelcmtd) [inline pages.de/common/aircrack-ng.md:7]:
```suggestion
- Knacke Schlüssel von abgefangenen Paketen mithilfe von Wortlisten:
```

File: pages.de/common/aircrack-ng.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+`aircrack-ng -w {{pfad/zu/wortliste.txt}} {{pfad/zu/packetdatei.cap}}`
+
+- Schlüssel knacken von abgefangenen Packeten mithilfe einer Wortliste und der (E)SSID des Access Points:
```
Comment (pixelcmtd) [inline pages.de/common/aircrack-ng.md:11]:
```suggestion
- Knacke Schlüssel von abgefangenen Paketen mithilfe einer Wortliste und der (E)SSID des Access Points:
```

File: pages.de/common/aircrack-ng.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+`aircrack-ng -w {{pfad/zu/wortliste.txt}} -e {{essid}} {{pfad/zu/packetdatei.cap}}`
+
+- Schlüssel knacken von abgefangenen Packeten mithilfe einer Wortliste und der MAC-Adresse des Access Points:
```
Comment (pixelcmtd) [inline pages.de/common/aircrack-ng.md:15]:
```suggestion
- Knacke Schlüssel von abgefangenen Paketen mithilfe einer Wortliste und der MAC-Adresse des Access Points:
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
Please check the [Style Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood) and a [dictionary](https://www.duden.de/rechtschreibung/Paket) _before_ contributing.

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**